service/version_bundles.go

```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/giantswarm/kvm-operator/service/controller/v23patch1"
 	v24 "github.com/giantswarm/kvm-operator/service/controller/v24"
 	v25 "github.com/giantswarm/kvm-operator/service/controller/v25"
+	v25 "github.com/giantswarm/kvm-operator/service/controller/v26"
```

Comment:
This should not even compile. 

```suggestion
	v26 "github.com/giantswarm/kvm-operator/service/controller/v26"
```